test/performance/odp_ipsec.c

```diff
@@ -809,6 +809,16 @@ run_measure_one_config(ipsec_args_t *cargs,
 		return -1;
 	}
 
+	if (cargs->schedule && !capa.queue_type_sched) {
+		app_err("Scheduled type destination queue not supported.\n");
+		return -1;
+	}
+
+	if (cargs->poll && !capa.queue_type_plain) {
```

Comment:
These checks are a bit too late since they are done after odp_ipsec_config() which sets the default queue.

Reply:
Ack. Will move the checks to main() itself.